### ISSUE TITLE
Serverside-rendering is broken on community and collection pages in watch mode

### ISSUE THREAD

**Issue by artlowel** [DSpace/dspace-angular#560]
If you open a community or collection page in watch mode with preboot enabled you get the following exception in the server output:

```
StaticInjectorError(ServerAppModule)[RouterlessTracking -> Router]: 
  StaticInjectorError(Platform: core)[RouterlessTracking -> Router]: 
    NullInjectorError: No provider for Router!
    at NullInjector../node_modules/@angular/core/fesm5/core.js.NullInjector.get (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40285:19)
    at resolveToken (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40523:24)
    at tryResolveToken (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40467:16)
    at StaticInjector../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40364:20)
    at resolveToken (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40523:24)
    at tryResolveToken (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40467:16)
    at StaticInjector../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:40364:20)
    at resolveNgModuleDep (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:47599:29)
    at _createClass (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:47646:29)
    at _createProviderInstance (/Users/art/IdeaProjects/dspace7-angular/code/dist/server.js:47616:26

```

The page will be empty apart from header and footer.

This seems to be due to Angulartics not being able to find the router, but they're both imported in [ServerAppModule](https://github.com/DSpace/dspace-angular/blob/ad77640f7d737984389ecbe5123d00d635e08cf5/src/modules/app/server-app.module.ts#L38-L50) where this error seems to originate from.

**Comment by Atmire-Kristof** [comment]
I'm experiencing the same issue. In my case, this was happening on the homepage, running the latest master branch.
At first, the error was shown in the server output without the page breaking. Everything else seemed to load like normal. After navigating the site for a while, coming back to the homepage breaks the page completely, showing an entire white screen (no header or footer in my case).
All other pages seemed to still load normally.

**Comment by tdonohue** [comment]
Fixed by #580
